tests/strictequal.test.js

```diff
@@ -0,0 +1,59 @@
+const describe = require('mocha').describe 
+
+const strictEqual = require ('../strictEqual')
+
+
+//make a test suite that covers entire file 
+const it = require('mocha').it
+
+const expect = require('chai').expect 
+
+
+//describe a test suite first parameter is a strig second is a function
+
+describe('strictEqual.js test suite', () =>{
+  describe ('strictEqual tests', ()=>{
+    it ('returns the value number and string', () => {
+      const mockData1 = 5
+      const mockData2 = '5'
+      const expectedResult = false 
+
+      const actualResult = strictEqual(mockData1,mockData2)
+
+      expect(actualResult).to.be.equal(expectedResult)
+    
+
+    }
+    )
+  }
+  )
+
+}
+
+
```

Comment:
Remove unnecessary whitespace and extra lines for cleaner code.

FOR EXAMPLE:

describe('strictEqual.js test suite', () => {
  describe('strictEqual tests', () => {
    it('returns the value number and string', () => {
      const mockData1 = 5
      const mockData2 = '5'
      const expectedResult = false

      const actualResult = strictEqual(mockData1, mockData2)

      expect(actualResult).to.be.equal(expectedResult)
    })
  })
})